src/cr/cube/matrix/measure.py

```diff
@@ -2391,10 +2421,14 @@ def _proportion_numerators(self):
         counts = self._second_order_measures.weighted_counts.blocks
 
         if self.orientation == MO.ROWS:
-            # --- Now get only the base values and relevant row subtotals
+            # --- Try getting the column comparable counts, to check for the error
+            self._second_order_measures.column_comparable_counts.blocks
```

Comment:
mmm this call doesn't do anything 🤷🏽‍♂️ 
it seems weird to me checking errors just calling a property